about/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: About the LLNL Software Portal
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+Welcome to the Lawrence Livermore National Laboratory software portal! The purpose of this software portal (or catalog if you prefer) is to serve as the definitive source of truth pointing to all software that has been produced and released by Lawrence Livermore National Laboratory.
```

Comment:
Longer term my intention was for this to cover both open source and other software we release. But I think that at least in the early phases, while it's mainly focused on open source, I'll update the wording to reflect that.